src/components/DownloadLink/DownloadLink.tsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, FlexItem, FlexProps } from '@patternfly/react-core';
+import { DownloadIcon } from '@patternfly/react-icons';
+
+interface IDownloadLinkProps {
+  url: string;
+  title: string;
+  showIcon?: boolean;
+}
+
+const spaceItemsSm: FlexProps['spaceItems'] = { default: 'spaceItemsSm' };
+
+export const DownloadLink = ({ url, title, showIcon = true }: IDownloadLinkProps) => {
+  return (
+    <Flex spaceItems={spaceItemsSm}>
+      {showIcon && (
+        <FlexItem>
+          <a href={url} target="_self">
+            <DownloadIcon />
+          </a>
+        </FlexItem>
+      )}
+      <FlexItem>
+        <a href={url} target="_self">
+          {title}
+        </a>
+      </FlexItem>
+    </Flex>
```

Comment:
the download icon and the link are doing the same job, if we already have the download link maybe consider using `CopyToClipboard` component if needed.

Reply:
I dont see an issue with it, and I don't find the 'copy to clipboard' feature that much needed here whatsoever.